src/reducers/index.js

```diff
@@ -1,8 +1,11 @@
-import { combineReducers } from 'redux';
-import { counters } from './counters';
-import { creatures } from './creatures';
+import {combineReducers} from 'redux';
+import {counters} from './counters';
+import {creatures} from './creatures';
+import {stateManagement} from './stateManagement';
 
-export default combineReducers({
+export default function (state, action) {
+  return stateManagement(state, action, combineReducers({
     counters,
-    creatures,
-});
+    creatures
+  }));
+}
```

Comment:
In this file we overwrite the normal redux reducer to be the stateManagement reducer instead of the normal redux combo-reducer. The stateManagement reducer accepts the normal combo-reducer so that it can delegate it -- maintaining normal operations for all actions except load state, which overwrites the entire redux state.